cells/lib/ops/revise.nix

```diff
@@ -0,0 +1,65 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs;
+  l = nixpkgs.lib // builtins;
+in
+  /*
+  For use with `revisePackage` and `reviseOCI` to build containers in a mono-repo style
+  environment where the source code is contained in the same repository as the std code,
+  specifically so that one may detect meaningful changes to the image via its tag in the
+  special case where the package's output includes the revision of the source code (e.g. for
+  displaying the version to the user).
+
+  Without special processing, this kind of package would cause the OCI image tag to change
```

Comment:
```
Explanation at the example of OCI Images:
```

To make it clear this specificity is at the service of an example, not a tacit narrowing of the api (as `revisePackage` was mentioned explicitly before).

Reply:
Or generalize the explanation since `reviseOCI` has the same parragraph...